anchore_engine/clients/grype_wrapper.py

```diff
@@ -866,19 +869,10 @@ def query_record_source_counts(self, use_staging: bool = False):
                 # Transform the results along with the last_synced timestamp for each result
                 output = []
                 for result in results:
-                    feed_group = str(result[0]).split(":", 1)
-                    if len(feed_group) != 2:
-                        logger.error(
-                            "Unable to process feed/group for record_source {}. Omitting from the response".format(
-                                feed_group
-                            )
-                        )
-                        continue
-
                     record_source = RecordSource(
                         count=result[1],
-                        feed=feed_group[0],
-                        group=feed_group[1],
+                        feed=VULNERABILITIES,
```

Comment:
@nightfurys  Before this change we were getting this value from the fist part of the record_source field, which AFAICT was always `vulnerabilities`. I've set that to this constant here to keep the functionality from changing, but is there something else we want to set this to? `grype`?

I'm assuming once we remove the legacy scanner this field will be removed, so maybe it doesn't matter so much right now.

Reply:
Curious what's in the record_source field now. I see that it's still in the table definition.

Reply:
Let's leave this at vulnerabilities for now

Reply:
Ack, will do, thanks.